.github/workflows/ubuntu.yml

```diff
@@ -52,13 +52,6 @@ jobs:
           sudo apt install ${{matrix.compiler.cxx}} -y
         fi
 
-    - name: Configure bazel
```

Comment:
This should be unnecessary for the images you are using. Bazelisk will utilise and install the bazel version specified by `.bazelversion`